apps/dashboard_reports/static/dashboard_reports/styles/style.css

```
@@ -0,0 +1,230 @@
+
+.footer {
+  position: fixed;
+  bottom: -20mm;
+  left: 0;
+  height: 10mm;
+  width: 100%;
+  content: "Page " counter(page) " of " counter(pages);
+}
+
+.content {
+  page-break-inside: auto;
```

Stylelint (17.12.0)

[error] Expected "page-break-inside" to be "break-inside" (property-no-deprecated)

(property-no-deprecated)